app/services/platforms/facebook.ts

```diff
@@ -261,29 +263,23 @@ export class FacebookService extends StatefulService<IFacebookServiceState>
     this.SET_STREAM_PROPERTIES(title, description, game);
     await this.postPage(facebookPageId);
     if (this.state.liveVideoId && game) {
-      const headers = this.getHeaders(this.state.activePage.access_token);
-      const data = { title, description, game_specs: { name: game } };
-      const request = new Request(`${this.apiBase}/${this.state.liveVideoId}`, {
-        headers,
-        method: 'POST',
-        body: JSON.stringify(data),
-      });
-      return fetch(request)
-        .then(handlePlatformResponse)
-        .then(() => true);
+      return platformRequest(
+        {
+          url: `${this.apiBase}/${this.state.liveVideoId}`,
+          method: 'POST',
+          body: JSON.stringify({ title, description, game_specs: { name: game } }),
+        },
+        this.state.activePage.access_token,
+      ).then(() => true);
     }
     return Promise.resolve(true);
   }
 
-  @requiresToken()
   async searchGames(searchString: string): Promise<IGame[]> {
     if (searchString.length < 2) return;
-    const url = `${this.apiBase}/v3.2/search?type=game&q=${searchString}`;
-    const headers = this.getHeaders();
-    const request = new Request(url, { headers, method: 'GET' });
-    return fetch(request)
-      .then(handlePlatformResponse)
-      .then((json: any) => json.data);
+    return platformAuthorizedRequest(
+      `${this.apiBase}/v3.2/search?type=game&q=${searchString}`,
+    ).then(json => json.data);
```

Comment:
it looks like we're using both old and new request formations here? i feel like it'd be better for them to be uniform across the service

Reply:
I refactored only methods with @requiredToken decorator. But yeah, it's better to refactor all methods